mods/gen4/formats-data.js

```diff
@@ -1588,7 +1588,7 @@ exports.BattleFormatsData = {
 		tier: "LC",
 	},
 	glalie: {
-		randomBattleMoves: ["spikes", "icebeam", "iceshard", "earthquake", "explosion", "taunt"],
```

Comment:
Personally I'd still keep Ice Shard Glalie is usually a suicide lead anyways so priority helps